students/BrandonHenson/Lesson9/Mailroom_5.py

```diff
@@ -0,0 +1,138 @@
+# Brandon Henson
+# 5/17/18
+# Lesson 09 mailroom_5.py
+# Refactoring
+
+
+class Donor(object):
+
+    donor_history = {'Brandon Henson': [1005.49, 3116.72, 5200],
+                        'Alicia Henson': [21.47, 1500],
+                        'Michael Green': [2400.54],
+                        'Brandon Henson Jr': [355.42, 579.31],
+                        'Kaiya Henson': [636.9, 850.13, 125.23]}
+
+    name_list = [i for i in donor_history]
+
+    def __init__(self, name, *donation):
```

Comment:
Don't think you need the `*` here, since this indicates a variable number of args